_posts/2023-08-04-deploying-OIDC-webapp-on-openshift.adoc

```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+title: 'Deploying an OpenID Secured WildFly Application on OpenShift'
```

Comment:
I think we should update the title to reflect that this post is about trying out a custom branch on OpenShift, maybe something like:

Try out a new in-progress feature for WildFly apps secured with OIDC on OpenShift

Reply:
The filename should also be updated to reflect this.

Reply:
fixed!